### PR TITLE
feat: added wishlist

### PR DESCRIPTION
**Pull request by jDineshKarthic**

### CODE REVIEW

File: components/product-listing/wishlistPage/rootwishlist.css [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    --lt-pink: #fff2f0;
+    --lt-red: #ffe7e7;
+    --dk-red: rgb(230, 86, 86);
+    --dk-red: rgb(224, 39, 39);
+    --lt-maroon: #dc9891;
+    --sm: 1rem;
+    --md: 1.5rem;
+    --xs: 0.5rem;
+
+    --semi-bold: 500;
```
Comment (ttahm3d) [inline components/product-listing/wishlistPage/rootwishlist.css:21]: If these are common across all the pages, include them in root and import them here

**Review by therajatg — APPROVED** [review]
looks good

File: components/product-listing/wishlistPage/wishlist.css [hunk 2]
```diff
@@ -0,0 +1,282 @@
...
+    font-family: 'Arima Madurai', cursive;
+    font-weight: 100;
+}
+
+.input-items {
+    padding: 0.6rem;
+    padding-right: 4rem;
+    border-radius: 1rem;
+    border: none;
+    /* color: var(--dk-grey); */
```
Comment (Namrata-J) [inline components/product-listing/wishlistPage/wishlist.css:54]: you can remove the commented code from here

File: components/product-listing/wishlistPage/wishlist.css [hunk 3]
```diff
@@ -0,0 +1,282 @@
...
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body {
+    background-color: var(--lt-cream);
+    font-family: 'Arima Madurai', cursive;
+
+}
```
Comment (ttahm3d) [inline components/product-listing/wishlistPage/wishlist.css:11]: This again is redandant. add it in reset files

File: components/product-listing/wishlistPage/wishlist.css [hunk 4]
```diff
@@ -0,0 +1,282 @@
...
+
+.filter-radio {
+    text-align: left;
+    padding: 0.25rem 0 0.25rem var(--md);
+}
+
+.clear-button-content {
+    text-align: center;
+    /* background-color: var(--lt-choco); */
```
Comment (Namrata-J) [inline components/product-listing/wishlistPage/wishlist.css:157]: remove commented code from everywhere

File: components/product-listing/wishlistPage/wishlist.css [hunk 5]
```diff
@@ -0,0 +1,282 @@
...
+    cursor: pointer;
+    position: relative;
+}
+
+.card {
+    height: max-content;
+    margin: var(--sm);
+    min-height: 25rem;
+    max-width: 17rem;
+    max-width: 350px;
```
Comment (Namrata-J) [inline components/product-listing/wishlistPage/wishlist.css:192]: you can try using rem for consistency

**Review by ttahm3d — COMMENTED** [review]
There is reduandant code from pervious PR. Check again and may be refactor if you have time

File: components/product-listing/wishlistPage/wishlist.css (hunk 3 above)
Comment (jDineshKarthic) [reply components/product-listing/wishlistPage/wishlist.css:11]: yeah i'll do that tomorrow

**Review by Namrata-J — APPROVED** [review]

**Review by aliamaanansari — COMMENTED** [review]
Looks Good !

**Pull request merged**